SDK_dotnet/Services/GluwaService.cs

```diff
@@ -63,5 +64,33 @@ internal static string getGluwacoinContractAddress(ECurrency currency, Environme
                     throw new ArgumentOutOfRangeException($"Unsupported currency: {currency}");
             }
         }
+
+        internal static string GetNonceString()
+        {
+            string nonceString = string.Empty;
+            UInt64 uInt64Nonce = 0;
+            do
+            {
+                uInt64Nonce = NextInt();
+                string stringNonce = uInt64Nonce.ToString();
+
+                nonceString += stringNonce;
+
+            } while (nonceString.Length < 75);
+
+            nonceString = nonceString.Substring(0, 75);
+
+            return nonceString;
+        }
+
+        internal static UInt64 NextInt()
+        {
+            RNGCryptoServiceProvider rng = new RNGCryptoServiceProvider();
+            byte[] buffer = new byte[8];
+            rng.GetBytes(buffer);
+            UInt64 result = BitConverter.ToUInt64(buffer, 0);
+
+            return result;
+        }
```

Comment:
`RNGCryptoServiceProvider` needs to be disposed. See https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rngcryptoserviceprovider?view=net-5.0#remarks